api/types.go

```diff
@@ -12,21 +15,31 @@ type ProxyConfig struct {
 }
 
 type OIDCConfig struct {
-	IssuerUrl    *string
-	ClientId     *string
-	ClientSecret *string
-	Audience     *string
+	IssuerUrl    *string `json:"issuer_url"`
```

Comment:
Accronym types in Go should be uppercased as a convention, except from when it affects exportability of the type.

This means that they should be named:
IssuerURL
ClientID